planning/rtc_auto_approver/config/rtc_auto_approver.param.yaml

```diff
@@ -0,0 +1,29 @@
+/**:
+  ros__parameters:
+    module_list:
+      - "behavior_velocity_planner/blind_spot"
+      - "behavior_velocity_planner/crosswalk"
+      - "behavior_velocity_planner/detection_area"
+      - "behavior_velocity_planner/intersection"
+      - "behavior_velocity_planner/no_stopping_area"
+      - "behavior_velocity_planner/traffic_light"
+      - "behavior_path_planner/lane_change_left"
+      - "behavior_path_planner/lane_change_right"
+      - "behavior_path_planner/avoidance_left"
+      - "behavior_path_planner/avoidance_right"
+      - "behavior_path_planner/pull_over"
+      - "behavior_path_planner/pull_out"
+
+    enable_list:
```

Comment:
If it can be changed dynamically, the name may be better to `defaut_enable_list`